docusaurus/docs/core-concepts/config-store/standard-config-type-intercept.v1.md

```diff
@@ -0,0 +1,50 @@
+
+# Standard Config Type intercept.v1
+
+A Config of Config Type `intercept.v1` is used by a Ziti tunneler application to configure itself as a proxy for a particular Ziti service.
+
+## Examples
+
+This Config instructs the tunneler to capture any outgoing traffic with destination `tcp:acme.example.ziti:5000`. A tunneler that is running in proxy mode provides a Ziti nameserver that resolves domain names to intercept IP addresses.
```

Comment:
"A tunneler that is running in proxy mode" --> "An intercepting tunneler" ?

"capture" --> "intercept" ?

"Ziti nameserver" --> "DNS nameserver" ?